modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerContainerFragment.kt

```diff
@@ -208,6 +258,35 @@ class PlayerContainerFragment : BaseFragment(), HasBackstack {
             menuRes = null,
             fragmentManager = parentFragmentManager,
         )
+
+
+        viewModel.networkType.observe(viewLifecycleOwner) { type ->
+            when (type) {
+
+                ConnectivityManager.TYPE_WIFI -> {
+                    Timber.tag("Check Data Connectivity").d("Connect to wifi 😊")
+                    println("Connect to wifi ")
+
+                }
+
+                ConnectivityManager.TYPE_MOBILE -> {
+                    //Log Purposes
+                    Timber.tag("Check Data Connectivity").d("Connect to Mobile Internet 😊")
+                    println("Connect to Mobile Internet ")
+                    //Toast when user is using Mobile Internet
+                    Toast.makeText(context, "Using Mobile Data", Toast.LENGTH_SHORT).show()
```

Comment:
For strings like this that are user facing, we need to use a string resource so that they get translated.